doc/sphinx/source/core/quantization.rst

```diff
@@ -0,0 +1,168 @@
+frameworks with fused ops to nGraph. Unlike builders, experimental ops take
+scale and zero point instead of min and max.
+
+.. table:: Experimental Quantized Ops (optional)
+
+
+	+-----------------------------------+-------------------------------------+
+	| Operator                          | Description                         |
+	+===================================+=====================================+
+	| QuantizedConvolutionBias          | This experimental op can be         |
+	|                                   | fused with a ReLU op.               |
+	+-----------------------------------+-------------------------------------+
+	| QuantizedConvolutionBiasAdd       | This experimental op constructs a   |
+	|                                   | quantized convolution with bias and |
+	|                                   | optional ReLU. And then takes input |
+	|                                   | for the add operation.              |
+	+-----------------------------------+-------------------------------------+
+	| QuantizedConvolutionBiasSignedAdd | Same as QuantizedConvolutionBiasAdd |
+	|                                   | but with signed add.                |
+	+-----------------------------------+-------------------------------------+
+	| QuantizedConvolutionRelu          | This experimental op is designed    |
+	|                                   | for a particular use case that      |
+	|                                   | would require convolution           |
+	|                                   | and ReLU to be combined.            |
+	+-----------------------------------+-------------------------------------+
+	| QuantizedDotBias                  | This experimental op can be fused   |
+	|                                   | with a ReLU op.                     |
+	+-----------------------------------+-------------------------------------+
+
+nGraph Quantization Design
+--------------------------
+
+The goal of nGraph quantization is to flexibly support a wide variety of
+frameworks and users. The use of scale and zero point as well as quantized
+builders in the nGraph design helps to achieve this goal.
+
+Scale and Zero Point
+~~~~~~~~~~~~~~~~~~~~
+
+Using scale and zero point allows nGraph to be framework agnostic (i.e., it
+can equally support all deep learning frameworks). nGraph Bridges will
+automatically convert min and max (provided by a DL framework) to scale and zero
+point as needed. Quantized builders are available to help the bridges perform
+this calculation. However, if users are directly using nGraph (and not using a
+bridge), they are required to provide scale and zero point for quantized ops.
+
+Another advantage of using scale and zero point to express quantization
+parameters is that users can flexibly implement quantized ops into various
+nGraph backends. When implementing quantized ops, nGraph backends can directly
+use scale and zero point instead of min and max.
```

Comment:
I think it is accurate to say that *all current* nGraph backends *will* directly use scale and zero point (and not min and max) to perform the quantized computation.  I'm trying to make the point that we have to convert from min/max to scale/zero at some point.  This justifies the API choice for scale/zero.